keylime/tpm/tpm_main.py

```diff
@@ -1358,7 +1358,7 @@ def readPCR(self, pcrval, hash_alg=None):
         if jsonout is None:
             raise Exception("Could not read YAML output of tpm2_pcrread.")
 
-        if hash_alg not in jsonout:
+        if not jsonout.get(hash_alg):
```

Comment:
I see, this is a limitation of the conversion, that keep stuff like `{"sha1": None, ...}`

Reply:
Right. And if you ask whether "sha1" is in the map, then it will say "yes".